src/main/java/bhs/devilbotz/subsystems/DriveTrain.java

```diff
@@ -577,7 +578,7 @@ private void updateOdometry() {
    * @since 1/30/2023
    */
   public double getRoll() {
-    return navx.getRoll();
+    return -navx.getRoll();
```

Comment:
navx on competition bot is rotated 180 compared to the practice bot, so the roll value needs to be negated. _This needs to be moved to the robotconfig json files as a flag.....e.g. rollInversion=true/false_